index.html

```diff
@@ -0,0 +1,97 @@
+        <h1>Pinchuk Vladislav Alexandrovich</h1>
+    </header>
+    <hr>
+    <img src="photo.jpg" width="150" alt="Photo Pinchuk Vladislav">
+     
+    <section>
+        <h3>Event Specialists / Event Manager <br>
+        <h4>Work experience - 11 years 8 months</h4>
+        </h3>
+        <p>Male, 30, born May 24, 1990</p>
+        <p><a href="tel:+375 (44) 515 49 83">+375 (44) 515 49 83</a><br>
+           <a href="mailto:freebka@gmail.com">freebka@gmail.com</a></p>
+        <p>Residence: Minsk <br>Citizenship: Belarus</p>
+    </section>
+
+    <section>
+        <h3>State institution Minskconcert</h3>
+        <p>July 2020 - present 3 months</p>
+        <h4>Head of the household</h4>
+        <article>Availability control in departments, purchase and delivery control of inventory items: drinking water, stationery, electrical household goods, household chemicals, consumables. Purchase of sound, light and computer equipment for the organization.
+            Inventory, accounting and write-off of material assets. Purchase of furniture, household and office equipment. Comprehensive organization of new jobs. Provision of personnel with the necessary equipment to carry out current work.</article>
+    </section>
+
+    <section>
+        <h3>Musical project “Peshahodka”</h3>
+        <p>May 2014 - present 6 years 5 months</p>
+        <h4>Director-Coordinator</h4>
+        <article>Leader of the "Peshahodka" project team: Directing, scripting, coordinating events, content creation for social networks of the project, video filming of project events, editing of short videos for social networks of the project, editing aftermovie,
+            booking artists, etc.</article>
+    </section>
+
+    <section>
+        <h4>Education - Higher</h4>
+        <p>(2013) <b>Belarusian State University of Culture and Arts</b> <br> Faculty of Traditional Belarusian Culture and Contemporary Art</p>
+    </section>
+
+    <section>
+        <h4>Key skills</h4>
+        <p>Knowledge of languages:<br> Belarusian - Native<br> English - A1 - Beginner<br> Russian - B2 - Intermediate advanced</p>
+        <p><b>Skills Project Management, Team Leadership, Teamwork, Creativity, PC User, Contracting, Speaking, Fast
+                Learning, Directing, Communication Skills, Organizational Skills, Event Organization, Presentation
+                Skills, Internal Communications, Event Management, Offsite Events , Leading groups in social networks,
+                Corporate events, Corporate culture, Videography, Video editing, MS PowerPoint.</b></p>
+    </section>
+
+    <section>
+        <h4>Additional Information</h4>
+        <h4>About me</h4>
+        <article>
+            <p>
+                Experience in event - 7 years.
+            <br> The skill of implementing an event on a turnkey basis - from writing a concept to actually holding it. Responsible and executive employee. I have experience in creating and leading a team for the implementation
```

Comment:
`<br` isn't needed here. The better solution could be usage of separating `<p>` tags

`<p>`content`<p>`
`<p>`content`<p>`